packages/auth/src/tests/signature.ts

```diff
@@ -78,10 +79,7 @@ export async function generateSigHeaders({
   let challenge = `"@method": ${method}\n"@target-uri": ${url}\n`
   let contentDigest
   if (body) {
-    const hash = crypto.createHash('sha256')
-    hash.update(Buffer.from(JSON.stringify(body)))
-    const bodyDigest = hash.digest()
-    contentDigest = `sha-256:${bodyDigest.toString('base64')}:`
+    contentDigest = createContentDigestHeader(JSON.stringify(body), ['sha-256'])
```

Comment:
```suggestion
    contentDigest = createContentDigestHeader(JSON.stringify(body), ['sha-512'])
```
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-7.3.1
>   When the proof method is specified in string form, the signing
   algorithm MUST be derived from the key material (such as using the
   JWS algorithm in a JWK formatted key), and the content digest
   algorithm MUST be sha-512.

That said, since we've switched to specifying clients by payment pointer, we now only have `proof` in the introspection response:
- https://github.com/interledger/rafiki/pull/739

I vote we follow the spirit of the spec and do sha512 when not otherwise specified.

Reply:
Works for me